README.md

```diff
@@ -30,28 +32,35 @@ $ terraform destroy
 ```
 
 ## Module Input Variables
+
 **Required**
-- `vpc` - VPC id for ECS cluster
-- `subnets` - List of subnet ids for ECS cluster, please choose 2 subnets
-- `key_name` - Name of key pair for SSH login to ECS cluster instances
+
+* `vpc` - VPC id for ECS cluster
+* `subnets` - List of subnet ids for ECS cluster, please choose 2 subnets
+* `key_name` - Name of key pair for SSH login to ECS cluster instances
 
 **Optional**
-- `ami` - ECS cluster instance AMI id, default is Amazon ECS-optimized AMI in `us-east-1`
-- `app_name` - Your application name, default is `demo-app`
-- `image` - Your docker image name, default it ECS PHP Simple App
-- `container_port` - Port number exposed by container, default is 80
-- `service_count` - Number of containers, default is 3
-- `cpu_unit` - Number of cpu_units for container, default is 128
-- `memory` - Number of memory for container, default is 128
-- `spot_prices` - Bid amount to spot fleet, please choose 2 prices. Default is `$0.03`
-- `strategy` - Instance placement strategy name, default is `diversified`
-- `instance_count` - Number of instances, default is 3
-- `instance_type` - Instance type launched by Spot Fleet, default is `m3.medium`
-- `volume_size` - Root volume size, default is 16
-- `app_port` - Port number of application, default is 80
-- `valid_until` - limit of Spot Fleet request, default is `2020-12-15T00:00:00Z`
+
+* `ami` - ECS cluster instance AMI id, default is Amazon ECS-optimized AMI in `us-east-1`
+* `app_name` - Your application name, default is `demo-app`
+* `image` - Your docker image name, default it ECS PHP Simple App
+* `container_port` - Port number exposed by container, default is 80
+* `service_count` - Number of containers, default is 3
+* `cpu_unit` - Number of cpu_units for container, default is 128
+* `memory` - Number of memory for container, default is 128
+* `spot_prices` - Bid amount to spot fleet, please choose 2 prices, default is `$0.03`
+* `strategy` - Instance placement strategy name, default is `diversified`
+* `instance_count` - Number of instances, default is 3
+* `instance_type` - Instance type launched by Spot Fleet. default is `m3.medium`
+* `volume_size` - Root volume size, default is 16
+* `plain_http` - Whether the load balancer should listen to plain http requests, default is `true`
+* `https` - Whether the load balancer should listen to https requests, default is `false`
```

Comment:
I guess this module should only accept `https`. If `https` is true, creates `aws_alb_listener.https`. If not, creates `aws_alb_listener.http`

Reply:
Sounds good !  I updated it to be one or the other.